firestore/query.go

```diff
@@ -43,6 +43,7 @@ type Query struct {
 	orders                 []order
 	offset                 int32
 	limit                  *wrappers.Int32Value
+	limitToLast            bool
```

Comment:
Using just a `bool` instead of describing limit types (like they did in Java and NodeJS) to save space